baseplate/context/memcache/__init__.py

```diff
@@ -5,11 +5,12 @@
 
 from pymemcache.client.base import PooledClient
 
-from . import ContextFactory
-from .. import config
+from .lib import make_memcache_serializer, memcache_deserializer
+from baseplate.context import ContextFactory
+from baseplate import config
 
 
-def pool_from_config(app_config, prefix="memcache.", **kwargs):
```

Comment:
are there no other options that might need to get passed through? it be nice to keep the **kwargs if there are other options (and perhaps even if there aren't so some config can be hard-coded when desired)